samples/ts/21.react-media-template/src/pages/TabConfig.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as microsoftTeams from "@microsoft/teams-js";
+import React, { FC, useEffect } from "react";
+import { getFlexColumnStyles } from "../styles/layouts";
+import { mergeClasses, Title2, Subtitle2 } from "@fluentui/react-components";
+
+const TabConfig: FC = () => {
+    useEffect(() => {
+        microsoftTeams.pages.config.registerOnSaveHandler(function (saveEvent) {
+            microsoftTeams.pages.config.setConfig({
+                suggestedDisplayName: "Contoso",
+                contentUrl: `${window.location.origin}/sidepanel?inTeams=true`,
+            });
+            saveEvent.notifySuccess();
+        });
+
+        microsoftTeams.pages.config.setValidityState(true);
+    }, []);
+
+    const flexColumnStyles = getFlexColumnStyles();
+    return (
+        <div
+            className={mergeClasses(
+                flexColumnStyles.root,
+                flexColumnStyles.hAlignCenter,
+                flexColumnStyles.vAlignCenter,
+                flexColumnStyles.fill,
+                flexColumnStyles.smallGap
+            )}
+        >
+            <Title2 block align="center">
+                Welcome to Contoso Media!
+            </Title2>
+            <Subtitle2 block align="center">
```

Comment:
I'm less sure about this one...
```suggestion
            <Subtitle2 as="p" block align="center">
```